scala/sbt/dse/src/main/scala/com/datastax/spark/example/WriteRead.scala

```diff
@@ -1,10 +1,9 @@
 package com.datastax.spark.example
 
-import org.apache.spark.sql.cassandra._
-import org.apache.spark.{SparkConf, SparkContext}
-
 import com.datastax.spark.connector._
 import com.datastax.spark.connector.cql.CassandraConnector
+import org.apache.spark.sql.hive.HiveContext
+import org.apache.spark.{SparkConf, SparkContext}
 
 object WriteRead extends App {
 
```

Comment:
May also want to add a note (just because we just got another support ticket about this) that in DSE it is not necessary to set an connection parameters for spark.master (since it will be done automatically)